data_models/README.md

```diff
@@ -205,6 +217,8 @@ Scraper authors, please keep an eye out for amendments to the data model.
 # Race and Ethnicity
 We need to collapse counties that report race and ethnicity into one race/ethnicity dimension. This section will be updated pending information about San Francisco County's methods for reporting race and ethnicity.
 
+The category "Multi_or_Other" was included because Marin rolls up the numbers from "Multi" and "Other" into one.
+
```

Comment:
Maybe add: "Please note that this category is not relevant for counties that report 'Mulitple Race' and 'Other Race' separately."
This is yet another sign that maybe we should just take the race category names exactly as the county reports them, like we're now doing with gender and age.

Reply:
Honestly, this makes a lot of sense to do. It would come with the bonus of making all the scrapers more resilient against changes to the reported races like what happened with Sonoma

Reply:
I'll add that Elaine - thank you both!